src/main/java/lessons/hw3/Solution.java

```diff
@@ -0,0 +1,59 @@
+package lessons.hw3;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Solution {
+
+    public static void main(String[] args) {
+        List<Grade> johnGrades = List.of(
+            new Grade("Language",88.5),
```

Comment:
```suggestion
            new Grade("Language", 88.5),
```
please configure checkstyle